src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -304,9 +304,10 @@ Emscripten_HandleMouseMove(int eventType, const EmscriptenMouseEvent *mouseEvent
     EmscriptenPointerlockChangeEvent pointerlock_status;
 
     /* check for pointer lock */
-    emscripten_get_pointerlock_status(&pointerlock_status);
+    int isPointerLockSupported = emscripten_get_pointerlock_status(&pointerlock_status);
+    int isPointerLocked = isPointerLockSupported == EMSCRIPTEN_RESULT_NOT_SUPPORTED ? SDL_FALSE : pointerlock_status.isActive;
```

Comment:
Since there are multiple error conditions that the HTML5 functions can return, I'd recommend testing here directly against `EMSCRIPTEN_RESULT_SUCCESS` (https://github.com/kripken/emscripten/blob/master/system/include/emscripten/html5.h#L60) instead of only one of the error conditions. Probably doesn't change anything in practice, but might be a bit more future-proof.